.golangci.yml

```diff
@@ -194,31 +195,11 @@ linters:
     - testpackage
     - thelper
     - tparallel
-    - typecheck
     - unconvert
     - unparam
     - unused
     - wastedassign
     - whitespace
-  enable-all: true
```

Comment:
iirc this is still needed for default linters to be enabled. Can you check the document for v1.46.0

Reply:
We have all the ones we want in the explicitly enabled list.  This gives us much more stability when we upgrade golangci-lint as we won't get nearly as many failures due to them changing the default linters.

Reply:
ack